spec/menu_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'menu'
+
+describe Menu do
+
+  it "Is able to store a list of dishes with name and price." do
+    subject.add_dish("Special", "8.50")
```

Comment:
This is where you could have a let double i.e.

let(:food_item) { double:menu }  or { "Special" }
let(:price) { double:menu } or { 8.50 } 

The bits in the {} are place holders so could be literally anything 
The let is referring to the parameter name